Nicanor008/shoman-FE
# Mentee application

## What's this PR about
Allows a mentee to send an application to join the Shoman platform.

## How to test this PR
- clone and open this project
- checkout to this branch `git checkout mentee-apply-#24`
- run development env `gatsby develop`, of course, you should have gatsby installed globally. if not, check Readme.md documentation.
- on the browser, open `http://localhost:4000/apply`

## Issue
#24 

## Output Screenshots
<img width="1680" alt="Screenshot 2020-12-09 at 19 49 38" src="https://user-images.githubusercontent.com/25298492/101660044-c8c74380-3a57-11eb-94a4-621ff180caa3.png">
